source/_integrations/huawei_lte.markdown

```diff
@@ -76,18 +77,22 @@ numbers can be set using the integration's configuration options.
 
 The following router action services are available. When invoked by a user, administrator access is required.
 
-### Service `huawei_lte.clear_traffic_statistics`
+### Deprecated service `huawei_lte.clear_traffic_statistics`
```

Comment:
Please remove deprecated documentation instead of marking it deprecated.

Reply:
Done. https://github.com/home-assistant/home-assistant.io/pull/27144 removes it from the fan docs whose example I followed.